packages/gallery/src/components/item/texts/lineHeightFixer.js

```diff
@@ -147,20 +147,21 @@ class LineHeightFixer {
         [PLACEMENTS.SHOW_ON_THE_LEFT]: 'gallery-item-left-info',
       }[styleParams.titlePlacement];
       const elements = itemContainer.getElementsByClassName(className);
-      const element = elements.length > 0 && elements[0];
-      const elementPadding =
-        parseInt(this.getCss(element, 'padding-top', 0)) +
-        parseInt(this.getCss(element, 'padding-bottom', 0));
-      const margin =
-        styleParams.imageInfoType === INFO_TYPE.SEPARATED_BACKGROUND &&
-        (styleParams.allowTitle || styleParams.allowDescription) &&
-        styleParams.textImageSpace
-          ? styleParams.textImageSpace
-          : 0;
-
-      const availableHeightOfTheTextElement = textPlacementAboveOrBelow ? options.externalTotalInfoHeight : textsContainerHeight; //maybe can be textsContainerHeight for textPlacementAboveOrBelow as well
-      availableHeight = availableHeightOfTheTextElement - elementPadding - margin;
-
+      if (typeof element !== 'undefined') {
```

Comment:
I mean, element (1, single) is defined 1 row below.
So, or do 'if (elements)', with an 's', to do the if 1 row below, after you define 'element' (not plural), and then.....